fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/provider/impl/AzureProvider.java

```diff
@@ -260,6 +271,9 @@ public void writeResources(String mediaType, List<ReadResultDTO> dtos) throws Ex
                 }
             }
 
+            ConfigurationAdapter config = ConfigurationFactory.getInstance();
+            long resourceCountThreshold = config.getCoreAzureObjectResourceCountThreshold();
+            long sizeThreshold = config.getCoreAzureObjectSizeThreshold();
```

Comment:
Is 0 supposed to be a special "no limit" value?  Based on the description in ConfigurationAdapter.java for these two config values.

Reply:
I left that off this particular one. I'll clean up the description in the adapter.

Reply:
Thanks!